lib/views/invoice_slack_view.ex

```diff
@@ -23,25 +25,63 @@ defmodule Aprb.Views.InvoiceSlackView do
     }
   end
 
+  defp invoice_transaction_message(event, partner_data) do
+    %{
+      text: ":oncoming_police_car: #{event["properties"]["seller_message"]}",
+      attachments: [%{
+        fields: [
+          %{
+            title: "Partner",
+            value: partner_data["name"],
+            short: true
+          },
+          %{
+            title: "Artworks",
+            value: artworks_display_from_artworkgroups(event["properties"]["invoice"]["artwork_groups"]),
```

Comment:
This might be a silly question. What happens if any of the nested keys missing? Do we need to make it more defensive?

Reply:
not a silly question at all, accessing it with `[]` will be fine since we get `nil` back but accessing it with dot notation will raise error
```elixir
iex(1)> a = %{a: '2', b: '3'}
%{a: '2', b: '3'}
iex(3)> a[:c][:d]
nil
iex(4)> a.c
** (KeyError) key :c not found in: %{a: '2', b: '3'}
iex(4)> a.a
'2'
```

Reply:
Thanks for the demo!! 👍